### PR TITLE
XCORNER-39 checkout styling

### PR DESCRIPTION
**Pull request by mgbelegu**
- imported styles on the optimized checkout
- extended styling to default checkout classes to match the design

### CODE REVIEW

File: assets/scss/optimized-checkout.scss [hunk 1]
```diff
@@ -1 +1,111 @@
...
+@import 'vendor/vendor';
+@import 'objects/objects';
+@import 'components/components';
+@import 'utilities/utilities';
+
+@mixin form-input() {
+    box-shadow: none;
+}
+
+.layout,
```
Comment (aleksandraworhacz) [inline assets/scss/optimized-checkout.scss:11]: Please add classname prefix if possible e.g. `.c-...`

File: assets/scss/optimized-checkout.scss [hunk 2]
```diff
@@ -1 +1,111 @@
...
+@import 'settings/settings';
+@import 'tools/tools';
+@import 'generic/generic';
+@import 'elements/elements';
+@import 'vendor/vendor';
+@import 'objects/objects';
+@import 'components/components';
+@import 'utilities/utilities';
+
+@mixin form-input() {
```
Comment (aleksandraworhacz) [inline assets/scss/optimized-checkout.scss]: Is it used in multiple places or just at line 40? If it's used in one place, you don't really need a mixin

**Review by aleksandraworhacz — COMMENTED** [review]
Added few comments

File: assets/scss/optimized-checkout.scss (hunk 1 above)
Comment (mgbelegu) [reply assets/scss/optimized-checkout.scss:11]: @aleksandraworhacz The checkout HTML content is rendered by BigCommerce and we have no control over it. We can't change the classnames or HTML.

(unless we use the BigCommerce Checkout SDK, but thats another story 😄  )

File: assets/scss/optimized-checkout.scss (hunk 2 above)
Comment (mgbelegu) [reply assets/scss/optimized-checkout.scss]: Done

**Comment by mgbelegu** [comment]
@aleksandraworhacz Can you please check again?

File: assets/scss/optimized-checkout.scss (hunk 1 above)
Comment (aleksandraworhacz) [reply assets/scss/optimized-checkout.scss:11]: I thought so, but it was worth to ask :)

**Review by aleksandraworhacz — APPROVED** [review]

**Pull request merged**